src/components/Card/CategoryCard/CategoryCard.jsx

```diff
@@ -1,24 +1,33 @@
-import "./CategoryCard";
+import "./CategoryCard.css";
 import { Link } from "react-router-dom";
+import { useFilter } from "../../../contexts";
 
 export const CategoryCard = ({ prodCat }) => {
+	const { filterDispatch: dispatch } = useFilter();
 	return (
 		<li
 			key={prodCat._id}
 			style={{ backgroundColor: "white" }}
-			className="categories__list"
+			className="categories__list category_card__list"
 		>
-			<Link to="/productlisting" className="categories__links">
+			<Link
+				onClick={() =>
+					dispatch({
+						type: "CATEGORY_SELECTION",
+						payload: prodCat.categoryName,
+					})
+				}
+				to="/productlisting"
+				className="categories__links"
```

Comment:
Suggestion : use Prettier to format the code

Reply:
i have 4 spaces as default spacing rule, it is formatted.